src/scss/index.scss

```diff
@@ -317,6 +317,12 @@ table.field-list {
         padding: 0.25rem 1.5rem;
         font-size: 0.9em;
         color: rgba(var(--color-sidebar-link), 1);
+
+        &:hover {
+          color: rgba(var(--color-sidebar-link-hover), 1);
```

Comment:
This is probably not the most robust way for doing this (also only works for 1 level of additional nesting), but since this is already done like that for the other style elements, this is the quick fix. 
But we should probably rework this part a bit more general.